packages/topkg-care/topkg-care.1.0.1/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "2.0"
+maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
+authors: ["Daniel Bünzli <daniel.buenzl i@erratique.ch>"]
+homepage: "https://erratique.ch/software/topkg"
+doc: "https://erratique.ch/software/topkg/doc"
+license: "ISC"
+dev-repo: "git+https://erratique.ch/repos/topkg.git"
+bug-reports: "https://github.com/dbuenzli/topkg/issues"
+tags: ["packaging" "ocamlbuild" "org:erratique"]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "ocamlfind" {build & >= "1.6.1"}
+  "ocamlbuild"
+  "topkg" {= "1.0.1"}
```

Comment:
just in case: you can use `= version` in opam (see https://opam.ocaml.org/doc/Manual.html#pkgvar-version) and will never have to adjust the version numbers again -- highly recommended (for libraries developed in the same repository where you don't plan to support different versions)

Reply:
Aha thanks.